### PR TITLE
Better styles for station list

### PR DESCRIPTION
**Pull request by pezholio**

### CODE REVIEW

File: config/stations.yml [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
   northbound: 870
+  class: interchange
 highbury_and_islington:
   southbound: 1035
   northbound: 1026
+  class: interchange
 kings_cross_st_pancras:
   southbound: 1193
   northbound: 1174
+  class: normal
```
Comment (pikesley) [inline config/stations.yml]: They're _all_ interchanges apart from Pimlico

Comment (pezholio) [inline config/stations.yml]: You're right actually - I was going by this https://tfl.gov.uk/tube/route/victoria/

File: public/style.css [hunk 2]
```diff
@@ -109,3 +109,55 @@ form {
 .wheel {
   font-size: 2em;
 }
+
+.victoria {
```
Comment (pikesley) [inline public/style.css:113]: Wait, what? Why does Victoria get special styling?

Comment (pezholio) [inline public/style.css:113]: That's the line, not the station :smile:

Comment (pikesley) [inline public/style.css:113]: Well don't I feel like a salted snail

**Comment by pikesley** [comment]
This is really good! Can we cap the ends somehow, so the blue line does not poke out of Brixton and Walthamstow?

File: public/style.css (hunk 2 above)
Comment (pezholio) [inline public/style.css:113]: It's perfectly OK to assume I'm going to do crazy CSS hacks like that :wink:

**Comment by pezholio** [comment]
Good call - I'll get on it

**Comment by pezholio** [comment]
Done! :+1:

**Pull request merged**